src/content/docs/new-relic-solutions/get-started/intro-new-relic2.mdx

```diff
@@ -0,0 +1,515 @@
+    alt="New Relic Kubernetes"
+    src={NRK8sMain}
+  />
+
+  </Side>
+  <Side>
+
+  The New Relic Kubernetes integration gives you full observability. The data it provides lets you monitor the health of your whole Kubernetes cluster and the performance of your environment. You can check individual pods or drill into specific services and applications. The agent collects telemetry data from your cluster. It uses several New Relic integrations, including Kubernetes events, the Prometheus Agent, and the New Relic Logs Kubernetes plugin.
+
+  Learn more about Kubernetes [here](l/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration).
+
+  </Side>
+  </SideBySide>
+
+# Live archives [#live-archives]
+<SideBySide>
+  <Side>
+
+  Live archives store historical logs with other logs and telemetry data in the New Relic Database. This allows instant access to old logs and stops the need to reload, re-index logs, or move data between many locations or tiers for analysis.
+
+  Learn more about live archives [here](/docs/logs/get-started/log-live-archives).
+
+  </Side>
+  <Side>
+
+  <img
+    title="New Relic Live archives"
+    alt="New Relic Live archives"
+    src={NRLiveArchivesMain}
+  />
+
+  </Side>
+  </SideBySide>
+
+# Logs [#logs]
+<SideBySide>
+  <Side>
+
+  <img
+    title="New Relic Live archives"
+    alt="New Relic Live archives"
+    src={NRLogsMain}
+  />
+
+  </Side>
+  <Side>
+
+  New Relic can help you manage large volumes of your log data. We provide tools to ingest only valuable logs to cut costs. We offer a unified UI to connect your <InlinePopover type="logs" /> to your services. We also offer tools that can help you sort through the clutter. 
+
+  Learn more about logs [here](/docs/logs/get-started/get-started-log-management/).
+
+  </Side>
+  </SideBySide>
+
+# Network [#network]
+<SideBySide>
+  <Side>
+
+  With <InlinePopover type="networkMonitoring" />, you can analyze how well your routers, switches, and other networking devices perform. It will also build a visual network topology, letting you find performance problems in your environment. 
+  
+  Learn more about network [here](/docs/network-performance-monitoring/get-started/npm-introduction/).
+
+  </Side>
+  <Side>
+
+<img
+  title="New Relic network"
+  alt="New Relic network"
+  src={NRNetworkMain}
+/>
+
+  </Side>
+  </SideBySide>
+
+# Mobile [#mobile]
+<SideBySide>
+  <Side>
+<img
+  title="New Relic mobile"
+  alt="New Relic mobile"
+  src={NRMobileMain}
+/>
+
+  </Side>
+  <Side>
+
+  The <InlinePopover type="mobile" /> capability sends telemetry directly from the end user's mobile app to New Relic. This allows us to analyze the app's performance on the user's device. This capability helps with the performance and crash troubleshooting of your Android, iOS, or hybrid mobile apps. Use mobile monitoring to improve your app's user experience. Or, study HTTP and network performance to work better with your backend teams.
```

Comment:
```suggestion
  The <InlinePopover type="mobile" /> capability sends telemetry directly from the end user's mobile app to New Relic. Mobile monitoring is flexible, and let you troubleshoot crashes from Android, iOS, or hybrid mobile apps. Study HTTP and network performance to work better with your backend teams.
```

Reply:
I'd unspool this sentence a bit more: Study HTTP and network performance to work better with your backend teams.

It sounds interesting and specific, something I might be interested in if i was a mobile developer. I think for me is I'm not sure what 'work better' 'means or what 'study' means in this context. Am I correlating data from HTTP and network performance so my backend teams can optimize a mobile app? Is it about how the mobile app itself communicates with HTTP requests over the network?

Reply:
Done!